eslint.config.mjs

```diff
@@ -0,0 +1,59 @@
+import Codex from "eslint-config-codex";
+import { plugin as TsPlugin, parser as TsParser } from 'typescript-eslint';
+
+export default [
+    ...Codex,
+    {
+        files: ['src/**/*.ts'],
+        languageOptions: {
+            parser: TsParser,
+            parserOptions: {
+                project: './tsconfig.json',
+                tsconfigRootDir: './',
+                sourceType: 'module',
+            },
+        },
+        rules: {
+            'n/no-missing-import': ['off'],
+            'n/no-unsupported-features/node-builtins': ['off'],
+            'jsdoc/require-returns-description': ['off'],
+            'jsdoc/require-jsdoc': [
+                'error',
+                {
+                    'require': {
+                        'FunctionDeclaration': true,
+                        'MethodDefinition': true,
+                        'ClassDeclaration': true,
+                        'ArrowFunctionExpression': false,
+                        'FunctionExpression': false,
+                    }
+                }
+            ],
+            '@typescript-eslint/explicit-member-accessibility': ['off'],
```

Comment:
seems unexpected

Reply:
Yes, I will remove it and provide modifiers to the required methods